Source/AssetRipper.GUI.Electron/Pages/Assets/View.cshtml.cs

```diff
@@ -60,6 +62,34 @@ static DirectBitmap TextureToBitmap(ITexture2D texture)
 			}
 		}
 
+		public byte[]? Font
+		{
+			get
+			{
+				return Asset is IFont font ? font.FontData_C128 : null;
+			}
+		}
+
+		public string? FontType
+		{
+			get
+			{
+				if (Font is { Length: >= 4 })
+				{
+					return (Font[0], Font[1], Font[2], Font[3]) switch
+					{
+						(0x4F, 0x54, 0x54, 0x4F) => "font/otf",    // "OTTO" in ASCII
```

Comment:
I would prefer these return just the file extension. In accordance with that, the property should be renamed.

Reply:
I think so too. How about modifying it to the following code?
```csharp
public string? FontFileName
{
	get
	{
		if (Asset is IFont font && Font is { Length: >= 4 })
		{
			return $"{font.GetBestName()}.{(Font[0], Font[1], Font[2], Font[3]) switch
			{
				(0x4F, 0x54, 0x54, 0x4F) => "otf",
				(0x00, 0x01, 0x00, 0x00) => "ttf",
				(0x74, 0x74, 0x63, 0x66) => "ttc",
				_ => null,
			}}";
		}
		return null;
	}
}
```